scripts/biomedical/chembl/README.md

```diff
@@ -0,0 +1,84 @@
+# Importing ChEMBL proteome data
+
+## Table of Contents
+
+- [Importing ChEMBL proteome data](#importing-chembl-proteome-data)
+  - [Table of Contents](#table-of-contents)
+  - [About the Dataset](#about-the-dataset)
+    - [Download URL](#download-url)
+    - [Overview](#overview)
+    - [Schema Overview](#schema-overview)
+    - [Notes and Caveats](#notes-and-caveats)
+    - [License](#license)
+  - [About the import](#about-the-import)
+  - [Examples](#examples)
+
+## About the Dataset
+
+### Download URL
+
+The ChEMBL database hosts a variety of datasets including, compound records, compounds, activities, assays, and targets. The ChEMBL data can be downloaded from the official [webpage](https://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/releases/chembl_29/). The data is in .txt and .fasta formats and had to be parsed into a csv version (see [Notes and Caveats](#notes-and-caveats) for additional information on formatting).
+
+### Overview
+
+This directory stores the scripts used to convert the datasets obtained from ChEMBL into modified versions, for effective ingestion of data into the Data Commons knowledge graph.
+
+The database links the ChEMBL ID, Uniprot ID, name, and sequence data of about various chemical compounds.
+
+### Schema Overview
+
+The schema representing chemical compound data from HMDB is defined in pre-defined. The tmcfs for each of the corresponding csv files can be found [here](https://github.com/suhana13/data/tree/add_hmdb_metabolites/scripts/biomedical/chembl/tmcf).
+
+### Notes and Caveats
+
+3 of the parsed files were in .fasta format. So, they had to be converted to a .csv format. However, the remaining two files were in .txt format and were easier to parse.
```

Comment:
Grammar error: never start a sentence with a number and always fully spell out a number that is ten or less